tmt-app/src/Components/MajorPage.jsx

```diff
@@ -1,7 +1,10 @@
 import React from "react";
+import { useParams } from "react-router-dom";
 
 function MajorPage() {
-  return <div>MajorPage</div>;
+  const majorChoice = useParams();
```

Comment:
this would be more clear as just `params` cause in the future, there could be more than just major related params. and `majorChoice.major` can be redundant